reporter/src/main/kotlin/model/EvaluatedModel.kt

```diff
@@ -95,18 +96,31 @@ import java.io.Writer
  *   Therefore [EvaluatedOrtIssue.pkg] is annotated with [JsonIdentityReference].
  */
 data class EvaluatedModel(
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val pathExcludes: List<PathExclude>,
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val scopeExcludes: List<ScopeExclude>,
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val copyrights: List<CopyrightStatement>,
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val licenses: List<LicenseId>,
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val scopes: List<EvaluatedScope>,
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val issueResolutions: List<IssueResolution>,
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val issues: List<EvaluatedOrtIssue>,
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val scanResults: List<EvaluatedScanResult>,
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val packages: List<EvaluatedPackage>,
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val paths: List<EvaluatedPackagePath>,
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val dependencyTrees: List<DependencyTreeNode>,
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val ruleViolationResolutions: List<RuleViolationResolution>,
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
```

Comment:
I would prefer not to exclude the values from the `EvaluatedModel`. They appear only once, so the effect on the file size is negligible, and I prefer that the generated model explicitly states that these lists are empty.

Regarding all other changes in this commit, did you test that they don't break the web app? Would it make the web app code more complex if it needs to check if those fields exist?

Reply:
1. Yes, done.
2. I did I quick smoke test and it seemed to work. Anyhow I don't know the WebApp very well and am not sure if I tested the right things. @tsteenbe do you think this change works out form WebApp perspective?

Reply:
@fviernau  I build the WebApp with the assumption the data it gets may not contain all fields so it should continue to work ..